Region/Histogram.cc

```diff
@@ -36,15 +36,15 @@ void Histogram::join(Histogram& h) { // NOLINT
     tbb::parallel_for(range, loop);
 }
 
-void Histogram::setup_bins(const size_t start, const size_t end) {
-    size_t i, stride, buckets;
+void Histogram::setup_bins() {
+    int64_t i, stride, buckets;
```

Comment:
while we're at it, can we change the name of one of the `i` variables, so that we don't have variable shadowing in the lamda?